agent/proxycfg/snapshot.go

```diff
@@ -5,8 +5,9 @@ import (
 	"fmt"
 	"sort"
 
-	"github.com/hashicorp/consul/agent/structs"
 	"github.com/mitchellh/copystructure"
+
+	"github.com/hashicorp/consul/agent/structs"
```

Comment:
Should this have moved?

Reply:
Yeah I picked this tweak up from @dnephin for invoking `goimports`:
```
let g:go_fmt_options = {
    \ 'goimports': '-local github.com/hashicorp/consul',
    \ }
```